Source/Shaders/ModelExperimental/LightingStageFS.glsl

```diff
@@ -7,16 +7,59 @@ vec3 LINEARtoSRGB(vec3 linearIn)
     #endif 
 }
 
+#ifdef LIGHTING_PBR
+vec3 applyTonemapping(vec3 linearIn) 
+{
+    #ifndef HDR 
+    return czm_acesTonemapping(linearIn);
+    #else 
+    return linearIn;
+    #endif 
+}
+
+vec3 computePbrLighting(czm_modelMaterial inputMaterial)
+{
+    czm_pbrParameters pbrParameters;
+    pbrParameters.diffuseColor = inputMaterial.diffuse;
+    pbrParameters.f0 = inputMaterial.specular;
+    pbrParameters.roughness = inputMaterial.roughness;
+
+    #ifndef USE_CUSTOM_LIGHT_COLOR
```

Comment:
We're currently not passing in a `lightColor` to `ModelExperimental` nor are we setting this define anywhere.

Reply:
Spoke to @lilleyse - we can add this feature at a later time.

Reply:
Removed the unused light color for now.